src/oviators/OviatorsExchangeMinterModule.sol

```diff
@@ -37,20 +37,32 @@ contract OviatorsExchangeMinterModule is
 
     event UpdatedColor(string color, ColorSetting settings);
     event UpdatedDescription(string newDescription);
+    event UpdatedImagesBase(string newBase);
+    event UpdatedRendererBase(string newBase);
 
     ERC721Drop public immutable source;
     ERC721Drop public immutable sink;
 
     string description;
+    string imagesBase;
+    string rendererBase;
```

Comment:
will these vars be readable in etherscan like source/sink are [here](https://basescan.org/address/0x9D1ECDF2D945141B004dD8405E00cBBAf29aC124#readContract)?  Do we need to add `public` for that?

Reply:
they don't have to be, since they are exposed via the `tokenURI` method...